src/layout/IHyprLayout.cpp

```diff
@@ -355,7 +355,7 @@ void IHyprLayout::onMouseMove(const Vector2D& mousePos) {
         canSkipUpdate = std::clamp(MSMONITOR - TIMERDELTA, 0.0, MSMONITOR) > totalMs * 1.0 / m_iMouseMoveEventCount;
     }
 
-    if ((abs(TICKDELTA.x) < 1.f && abs(TICKDELTA.y) < 1.f) || (TIMERDELTA < MSMONITOR && canSkipUpdate))
+    if (((abs(TICKDELTA.x) < 1.f && abs(TICKDELTA.y) < 1.f) || (TIMERDELTA < MSMONITOR && canSkipUpdate)) && g_pInputManager->dragMode != MBIND_MOVE)
```

Comment:
actually, no need to update if tickdelta is less than 1,1, so can you leave that in?

Reply:
wdym?

Reply:
if we are moving, rn, even if delta.x < 1 and delta.y < 1 we still process it, which is redundant

Reply:
tldr:
```cpp
if (((abs(TICKDELTA.x) < 1.f && abs(TICKDELTA.y) < 1.f) || (TIMERDELTA < MSMONITOR && canSkipUpdate)) && g_pInputManager->dragMode != MBIND_MOVE)
```
into 
```cpp
if ((abs(TICKDELTA.x) < 1.f && abs(TICKDELTA.y) < 1.f) || (TIMERDELTA < MSMONITOR && canSkipUpdate && g_pInputManager->dragMode != MBIND_MOVE))
```

Reply:
Done